react/ProductSummary.js

```diff
@@ -154,23 +187,23 @@ ProductSummary.schema = {
   properties: {
     showListPrice: {
       type: 'boolean',
-      title: 'Show product\'s list price',
+      title: "Show product's list price",
     },
     showLabels: {
       type: 'boolean',
-      title: 'Show product\'s prices\' labels (if false, shows only the prices)',
+      title: "Show product's prices' labels (if false, shows only the prices)",
     },
     showInstallments: {
       type: 'boolean',
-      title: 'Show product\'s payment installments',
+      title: "Show product's payment installments",
     },
     showBadge: {
       type: 'boolean',
       title: 'Show the discount badge',
     },
     badgeText: {
       type: 'string',
-      title: 'Badge\'s text',
+      title: "Badge's text",
     },
     hideBuyButton: {
```

Comment:
Add a prop to edit text label of the buy button. Something like `editButtonLabel`